javascript/node/selenium-webdriver/test/chrome/devtools_test.js

```diff
@@ -156,7 +156,6 @@ test.suite(
         await driver.get(fileServer.Pages.basicAuth)
         let source = await driver.getPageSource()
         assert.strictEqual(source.includes('Access granted!'), true)
-        await server.stop()
```

Comment:
Is this not required?

Reply:
Hi @corevo,

There is no server started in this test and have no import for this. 
AFAIK the server start and stop is managed by index.js in here https://github.com/JS31096/selenium/blob/b135bec7bfaf9f1081470602b6ca2b112130e602/javascript/node/selenium-webdriver/lib/test/index.js#L71